core/rend/gui.cpp

```diff
@@ -2423,6 +2423,9 @@ static void gui_display_settings()
 	            OptionCheckbox("Serial Console", config::SerialConsole,
 	            		"Dump the Dreamcast serial console to stdout");
 #endif
+				OptionCheckbox("Dreamcast 32MB RAM Mod", config::RamMod32MB,
+	            		"Enables 32MB RAM Mod for Dreamcast. May affect compatibility");
```

Comment:
I'm not sure if we should follow the weird tab spacing 😂

Reply:
There seems to be a combination of both tabs & spaces in this file. Which are preferred? Other files seem to be tabs but just wanted to confirm

Reply:
yes normally i follow the tabs & spaces...
@flyinghead what's your call?

Reply:
tabs are preferred but no big deal